### PR TITLE
feat: add ability to write logs in a file

### PR DESCRIPTION
**Pull request by shotonoff**
## Issue being fixed or feature implemented

By default the logs are written into stderr only. But, we encountered with the case when elastic doesn't have a bunch of the logs, so additionally we need to write the logs into a file along with writing its into stderr output.

## What was done?

Added ability to write the logs into the file along with stderr

## How Has This Been Tested?

N/A

## Breaking Changes

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: cmd/tenderdash/main.go [hunk 1]
```diff
@@ -64,3 +67,30 @@ func main() {
...
+	)
+	if conf.LogFilePath != "" {
+		file, err := os.OpenFile(conf.LogFilePath, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+		if err != nil {
+			return nil, nil, fmt.Errorf("failed to create log writer: %w", err)
+		}
+		closeFunc = func() {
+			_ = file.Close()
+		}
+		writer = io.MultiWriter(writer, file)
```
Comment (shumkov) [inline cmd/tenderdash/main.go:85]: I just wondering how it works. Does it flush every line into file or it's optimized for logging and has some buffer and flushing logic to do not introduce too much overhead?

Comment (shotonoff) [reply cmd/tenderdash/main.go:85]: this is a documentation
```
// MultiWriter creates a writer that duplicates its writes to all the
// provided writers, similar to the Unix tee(1) command.
//
// Each write is written to each listed writer, one at a time.
// If a listed writer returns an error, that overall write operation
// stops and returns the error; it does not continue down the list.
```

Comment (shumkov) [reply cmd/tenderdash/main.go:85]: Yeah, but does it have some buffer and flush logic? @lklimek

Comment (shotonoff) [reply cmd/tenderdash/main.go:85]: no, it doesn't use buffer

**Pull request merged**